### DISCUSSION TITLE
Tidak Dapat mengirim Pesan

### DISCUSSION THREAD

**Discussion by rizqifauzan** [kirimwa-id/forum#172] [Tanya Jawab]
Pesan error 
Can't send message. Error: TypeError: Cannot destructure property 'user' of '(0 , WABinary_1.jidDecode)(...)' as it is undefined

Device : iphone 8, Samsung M52

**Comment by kafaa-dev** [comment]
Errornya mirip #121 coba cek payloadnya sudah betul atau belum, mungkin bisa dilampirkan dengan menyensor informasi sensitif seperti nomor hp dan token.
